Gulpfile.js

```diff
@@ -171,6 +170,37 @@ gulp.task("build-check-test", gulp.series(
 gulp.task("build-lint-test", gulp.task("build-check-test"));
 
 // Helper functions:
+function spawnTask(command, done, args)
+{
+    let lCP;
+    if (args !== undefined)
+    {
+        lCP = spawn(command, args, { stdio: "inherit" });
+    } 
+    else
+    {
+        lCP = spawn(command, { stdio: "inherit" });
+    }
+    
+    lCP.on("error", (err) =>
+    {
+        console.error("Failed to start subprocess.");
+        done(err);
+    });
+
+    // catch non-zero exit statii so that CI can understand when task fails
+    lCP.on("close", (code, signal) => {
+        if (code !== 0)
+        {
+            done(new Error(`Task failed with error code ${code}`));
```

Comment:
we could just do `lCP.on("close", done)` but this Prettifies the output when tasks error.